tests/support/test.tcl

```diff
@@ -141,6 +141,31 @@ proc wait_for_condition {maxtries delay e _else_ elsescript} {
     }
 }
 
+proc verify_replica_online {master replica_idx max_retry} {
```

Comment:
again I prefer not to have each test take different max_retry. also maybe consider if just setting some time argument is not better in this case? I imagine we do not need to keep explicit time tracking but generally just divide the input by the sleep time (in your case 100)

Reply:
I agree in most cases setting a default timeout would work. However I didn't write this function (this is a refactor from `replication-psync.tcl`). For the existing tests, we need to be able to configure max retries. 
Regarding changing from max-retries to seconds, we already use the max-retries terminology in other places (`wait_for_condition` for example), using seconds will be confusing.